### ISSUE TITLE
Ubuntu 16.04 PHP 7

### ISSUE THREAD

**Issue by rperdon** [maxmind/GeoIP2-php#87]
Composer install does not work.

ler@Geoip-Dev-Web:/var/www/uploads$ php composer.phar require geoip2/geoip2:~2.0
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - The requested package geoip2/geoip2 No version set (parsed as 1.0.0) is satisfiable by geoip2/geoip2[No version set (parsed as 1.0.0)] but these conflict with your requirements or minimum-stability.

Installation failed, reverting ./composer.json to its original content.

**Comment by oschwald** [comment]
This works fine for me with the latest Composer version on Ubuntu with PHP 7. What Composer version are you using?

**Comment by rperdon** [comment]
I downloaded the most recent composer from here https://getcomposer.org/download/. I am running on Linux Ubuntu 16 Server with PHP 7.0. I was able to get it up and running via the GeoIP phar file but composer install was giving me a whole host of issues with regards  curl, php cli errors etc. Eventually I was left with the error I posted above which stumped me so I moved on to just running it off the phar file.
